### PR TITLE
Chore/#817 qg checks release 24 08

### PR DESCRIPTION
**Pull request by mkanal**
## Description
- various adjustments regarding #817 qg checks release 24 08

## Pre-review checks

Please ensure to do as many of the following checks as possible, before asking for committer review:

- [x] DEPENDENCIES are up-to-date. [Dash license tool](https://github.com/eclipse/dash-licenses). Committers can open IP issues for restricted libs.
- [x] [Copyright and license header](https://eclipse-tractusx.github.io/docs/release/trg-7/trg-7-02) are present on all affected files

### CODE REVIEW

File: SECURITY.md [hunk 1]
```diff
@@ -11,9 +11,13 @@ __How?__ Inside affected repository --> security tab
 for contributor:
 --> Report a vulnerability
 
+Open a draft security advisory : https://github.com/eclipse-tractusx/sig-release/security/advisories/new
+
 for committer:
 --> advisories --> New draft security advisory
 
+Open a draft security advisory : https://github.com/eclipse-tractusx/sig-release/security/advisories/new
+
```
Comment (ds-jhartmann) [inline SECURITY.md:20]: Why the change? TRG 7.01 shows how the content of the SECURITY.md file should look like and the current state is correct:
https://eclipse-tractusx.github.io/docs/release/trg-7/trg-7-01#security-file

File: .github/workflows/irs-build.yml [hunk 2]
```diff
@@ -110,7 +110,7 @@ jobs:
       - uses: actions/checkout@v4
 
       # Needed to create multi-platform image
-      - name: Set up Docker Buildx
+      - name: Set up Docker Build
```
Comment (ds-jhartmann) [inline .github/workflows/irs-build.yml:113]:
```suggestion
      - name: Set up Docker Buildx
```
"Buildx" is the correct name for the docker BuildKit capabilities

File: USAGE.md [hunk 3]
```diff
@@ -18,10 +18,15 @@ curl -X 'POST' \
   "aspects": [
```
Comment (ds-jhartmann) [inline USAGE.md:18]: I noticed, that the Authorization header is also outdated. Please adjust L16 to:
`-H 'X-API-KEY: <api_key>' \`

**Review by ds-jhartmann — CHANGES_REQUESTED** [review]

File: USAGE.md (hunk 3 above)
Comment (mkanal) [reply USAGE.md:18]: Please do so - if this is necessary. Thank you.

File: SECURITY.md (hunk 1 above)
Comment (mkanal) [reply SECURITY.md:20]: This is the way how CVS are reported in Tractus-X. So this link is important to be added to enable external contributors to reports CVS in the correct way.

Comment (ds-jhartmann) [reply SECURITY.md:20]: that's right, but the link points to the sig-release project, which is not correct and recommends to open a advisory even for contributors which is not possible. 
The recommended security file already contains all necessary information, so IMO we should not change this

**Review by ds-jhartmann — APPROVED** [review]
LGTM
